src/components/NewCommentForm.tsx

```diff
@@ -14,24 +63,31 @@ export const NewCommentForm: React.FC = () => {
             name="name"
             id="comment-author-name"
             placeholder="Name Surname"
-            className="input is-danger"
+            className={classNames('input', { 'is-danger': !userName })}
+            required
+            value={userName}
+            onChange={(event) => setUserName(event.target.value)}
           />
 
           <span className="icon is-small is-left">
             <i className="fas fa-user" />
           </span>
 
           <span
-            className="icon is-small is-right has-text-danger"
+            className={classNames(
+              'icon', 'is-small', 'is-right', { 'has-text-danger': !userName },
+            )}
             data-cy="ErrorIcon"
           >
             <i className="fas fa-exclamation-triangle" />
           </span>
         </div>
 
-        <p className="help is-danger" data-cy="ErrorMessage">
-          Name is required
-        </p>
+        {!userName && (
```

Comment:
you have empty fields by default, but you don't want to show the user an error before he even typed

<img width="553" alt="image" src="https://user-images.githubusercontent.com/55802630/187654456-58aef543-cea9-4dbd-b21b-2a26c18bca0b.png">

let's mb add some flag, like `isFilled` and set it in `onBlur`. And you won't show error if user hasn't touched this field yet

Reply:
fixed